syntax = "proto3";

package test_fdbrpc;

service TestEchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}

message EchoRequest {
    string message = 1;
}

message EchoResponse {
    string message = 1;
}